staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles/options.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package profiles
```

Comment:
It's a bit subjective, but packages tend to be singular, and I think it reads a bit better (e.g. `profile.Legacy`, `profile.Unprivileged`)